e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js

```diff
@@ -11,19 +11,27 @@ const { CURRENCY_CODE } = INSURANCE_FIELD_IDS.CURRENCY;
  * @param {String} isoCode: isoCode provided for radio selection
  * @param {Boolean} alternativeCurrency: if alternative currency should be entered
  */
-const completeAndSubmitAlternativeCurrencyForm = ({ isoCode, alternativeCurrency }) => {
+const completeAndSubmitAlternativeCurrencyForm = ({ isoCode, alternativeCurrency, clickAlternativeCurrencyLink = true }) => {
   if (isoCode) {
-    cy.clickProvideAlternativeCurrencyLink();
+    if (clickAlternativeCurrencyLink) {
+      cy.clickProvideAlternativeCurrencyLink();
+    }
 
     radios(CURRENCY_CODE, isoCode).option.label().click();
     cy.clickSubmitButton();
   }
 
   if (alternativeCurrency) {
-    cy.clickProvideAlternativeCurrencyLink();
+    if (clickAlternativeCurrencyLink) {
+      cy.clickProvideAlternativeCurrencyLink();
```

Comment:
Are all usages of this temporary until the new form is fully working?

Reply:
yes, once all the pages have the alternate currency removed,  then I can rework this slightly.  clickAlternativeCurrencyLink will be removed